README.md

```diff
@@ -1 +1,188 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [Hello Microverse Project Two] <a name="about-project"></a>
+
+
+**[Hello Microverse Project Two]** is a project on use of linters that only focuses on two languages which are html/css at the moment but could be modified later.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html/css.org/">Html/css</a></li>
+  </ul>
+</details>
+
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Added index.html]**
+- **[Better UI/UX Style]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Coming soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Web browser
+
+2. Code editor(visual studio code)
+
+3. Git/Github(version control system)
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1.git clone <the ssh url of the given repository>
+
+2.cd <the name of the repository>
+
+
+
+### Install
+
+Install this project with:
+
+To run the project on local machine execute the following command
+
+1.code .(to open on visual studio code)
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+1.Open index.html file on your visual studio then use live server to open it on your prefered browser.
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [Brenda Lutiku]
+- Twitter: [Brenda lutiku]
+- LinkedIn: [Brenda Lutiku]
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Javascript language]**
+- [ ] **[More style]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and the entire team including my fellow micronaunts for their support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Hey @brendalutiku, this is still not linking to your license file correctly. When it is clicked, it directs to a 404 page and i noticed that the problem is because the file is named `LICENCE` but here it is `LICENSE`, if you can fix the spelling issue, i believe it will all be fine and the readme will be very professional and satisfy all the requirement.

NOTE: When you are done, test the link and ensure it is not routing to a 404 page as see when this [link](https://github.com/brendalutiku/Hello-Microverse-Project-Two/blob/html-css-branch/LICENSE) is clicked but to the license file itself